pilot/pkg/config/monitor/copilot_snapshot.go

```diff
@@ -111,50 +112,91 @@ func (c *CopilotSnapshot) ReadConfigFiles() ([]*model.Config, error) {
 				},
 			},
 		}
+
+		if route.GetPath() != "" {
+			vs := config.Spec.(*networking.VirtualService)
+
+			vs.Http[0].Match = []*networking.HTTPMatchRequest{
+				{
+					Uri: &networking.StringMatch{
+						MatchType: &networking.StringMatch_Prefix{
+							Prefix: route.GetPath(),
+						},
+					},
+				},
+			}
+			vs.Http[0].Route[0].Destination.Subset = cfHash
+
+			rule := &model.Config{
+				ConfigMeta: model.ConfigMeta{
+					Type:    model.DestinationRule.Type,
+					Version: model.DestinationRule.Version,
+					Name:    fmt.Sprintf("rule-for-%s", route.GetHostname()),
+				},
+				Spec: &networking.DestinationRule{
+					Host: route.GetHostname(),
+					Subsets: []*networking.Subset{
+						{
+							Name:   cfHash,
+							Labels: map[string]string{"cf-service-instance": cfHash},
```

Comment:
why is this called `cf-service-instance`?  The `hash` is of a CF Route and CF Route Context Path.  That is only ever as granular as a single CF App.  Its not a CF App Instance, nor is it an Istio Service Instance.  

cc @zachgersh @utako